x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -66,10 +66,8 @@ const SavedQueryName = ({ savedQueryName }: SavedQueryNameProps) => (
   </EuiText>
 );
 
-const EuiBadgeWrap = styled(EuiBadge)`
-  .euiBadge__text {
-    white-space: pre-wrap !important;
-  }
+const FiltersWidthLimiter = styled.div`
```

Comment:
While Styled Components are legitimate to be used Emotion is preferred (for example see this [PR](https://github.com/elastic/kibana/pull/174867)).  EUI is migrating to Emotion and usually new code added to Security Solution uses Emotion as well. Usage is the following

```ts
import { css } from '@emotion/css';

...

const filtersWidthLimiter = css`
  max-width: 600px;
`;

...

<EuiFlexGroup className={filtersWidthLimiter} wrap responsive={false} gutterSize="xs" data-test-subj={dataTestSubj}>
```

Reply:
@maximpn I really liked the styling pattern used in https://github.com/elastic/kibana/pull/174867 👍 
Thank you for sharing this. Addressed.